mirror_lkl_layer.md

```diff
@@ -0,0 +1,53 @@
+## Mirror Last Known Locations layer (LKL) into a standard feature service
+
+This script will copy data from a Last Known Locations (LKL) layer in a Location Tracking (either Location Tracking Service or Location Tracking View) layer and mirror it into a standard feature service. 
+
+As of April 2020, the LKL layer does not support dynamic joins, so the mirroring of data using this script may be useful if the user is trying to relate additional data to their last known locations data. Standard feature services support join features.
+
+This script is designed to be run as a scheduled task, at whatever frequency you want to get updated LKL data from your Location Tracking layer. For example, the admin may set up this script to run every 10 minutes in order to get the latest data from the field.
+
+For information on how to set up scheduled tasks, please see: https://www.esri.com/arcgis-blog/products/arcgis-pro/analytics/schedule-a-python-script-or-model-to-run-at-a-prescribed-time-2019-update/
+
+and https://www.esri.com/arcgis-blog/products/product/analytics/scheduling-a-python-script-or-model-to-run-at-a-prescribed-time/
+
+This script requires the use of a point layer that the data can be mirrored into. This layer would ideally share all the same fields as a standard LKL layer, but could have less fields if all data is not requred. You can clone your Location Tracking Service (LTS) or Location Tracking View (LTV) using the `clone_items` functionality in the Python API.
```

Comment:
You should only clone the main service, if you try to clone the view, it will clone the view and the main service since the main service is a dependency.